src/components/live/index.tsx

```diff
@@ -20,6 +21,13 @@ const Live = () => {
       <Team />
       <Committees />
       <Judges />
+      <Countdown
```

Comment:
call it within the landing, since it will be integrated with other things within the landing